lib/Handler/FileHandler.php

```diff
@@ -33,34 +33,16 @@ public function __construct($minimalLevel, $verbose, $additionalParameters = arr
     /**
      * {@inheritDoc}
      */
-    public function write($level, $message, array $splitFullMessage, array $context = array())
+    public function write($level, $message, array $context = array())
     {
-        $i = 1;
-        foreach ($splitFullMessage as $fullMessage) {
-            $shortMessageToSend = $message;
-            if (count($splitFullMessage) != 1) {
-                $shortMessageToSend = $i.'/'.count($splitFullMessage).' '.$message;
-            }
-
-            $line = sprintf('%s %s: %s'.PHP_EOL, date('H:i:s'), strtoupper($level), $shortMessageToSend);
-            $this->writeToFile($line);
+        $line = sprintf('%s %s: %s'.PHP_EOL, date('H:i:s'), strtoupper($level), $message);
```

Comment:
It's strange to me to use sprintf and then concatenate a value via the concatenation operator. I would do
```
$line = sprintf('%s %s: %s%s', date('H:i:s'), strtoupper($level), $message, PHP_EOL);
```